pkgs/games/anki/default.nix

```diff
@@ -112,16 +112,10 @@ buildPythonApplication rec {
     ./no-version-check.patch
   ];
 
-  buildPhase = ''
-    # Dummy build phase
-    # Anki does not use setup.py
-  '';
+  # Anki does not use setup.py
+  dontBuild = true;
 
   postPatch = ''
-    # Remove unused starter. We'll create our own, minimalistic,
-    # starter.
-    # rm anki/anki
```

Comment:
Is this not relevant anymore?

Reply:
It was commented out back in 2018 and we seem to have been fine without it, so I'd say so:
https://github.com/NixOS/nixpkgs/commit/a30e30432da95d6eae8713b57ca5708d892a3263